docs/admin/compatibility_matrix.md

```diff
@@ -17,4 +17,4 @@ While that last supported version will not be strictly enforced via the `max_ver
 | 1.4.X                 | 1.5.3                          | 1.5.99 [Official]             |
 | 1.5.X                 | 1.6.1                          | 1.6.99 [Official]             |
 | 1.6.X                 | 1.6.1                          | 1.6.99 [Official]             |
-| 2.0.x                 | 2.0.0                          | TBD                           |
+| 2.0.x                 | 2.0.0                          | TBD !                         |
```

Comment:
Fix TBD pls. @itdependsnetworks @jeffkala

Reply:
Got it for the next commit I push.